deploy/crds/aws.managed.openshift.io_accountpools_crd.yaml

```diff
@@ -22,11 +22,12 @@ spec:
     listKind: AccountPoolList
     plural: accountpools
     singular: accountpool
-  scope: Namespaced
+  scope: Cluster
```

Comment:
Likewise, I thought accountpool was scoped to the aws-account-operator namespace.